server/index.js

```diff
@@ -4,32 +4,29 @@ const app = express();
 
 const cors = require("cors");
 const port = process.env.PORT || 5005;
-const pool = require("./database/db");
+const db = require("./database/db");
```

Comment:
Hi Kristina,
May I ask why you changed 'pool' to 'db'? I think things were working fine before. Let's maybe keep those things and not change them, please.

Reply:
Hi Elena, thanks for your comment. My renaming wouldn't change much. But if we already have started this way, you're right, better leave it how it was